tox.ini

```diff
@@ -324,3 +324,21 @@ commands =
 # https://github.com/sunpy/sunpy/pull/7432
     pytest dkist --benchmark-skip \
         -W "ignore::asdf.exceptions.AsdfManifestURIMismatchWarning"
+
+[testenv:abacusutils]
+change_dir = {env_tmp_dir}
+allowlist_externals =
+    git
+    bash
+extras =
+commands_pre =
+    bash -c "pip freeze -q | grep 'asdf @' > {env_tmp_dir}/requirements.txt"
+    git clone https://github.com/abacusorg/abacusutils.git
+    pip install -vU setuptools wheel scipy Cython 'numpy<2'  # for classy and corrfunc
+    pip install --no-build-isolation classy corrfunc
+    pip install -e abacusutils[test]
+    pip install -r {env_tmp_dir}/requirements.txt
+    bash -c "echo '' > pytest.ini"
```

Comment:
I had to make a blank "pytest.ini" here to prevent pytest from searching the parent directory and finding the asdf pytest configuration (in pyproject.toml in the asdf checkout). I think this should still work if abacusutils adds a pytest configuration (since that should be found before the blank pytest.ini created here).